charts/outline/values.yaml

```diff
@@ -39,6 +39,8 @@ replicaCount: 1
 ## @param outline.service.url The URL where the application will be accessible.
 ## @param outline.service.port The port on which the application will run.
 ## @param outline.fileStorage.type Type of file storage to be used (e.g., local, s3).
+## @param outline.fileStorage.storageClassName The storageclass name to use default "".
+## @param outline.fileStorage.storageSize The storageclass size to use default 250Gi.
```

Comment:
Did you use the `bitnami-helm-readme-generator` for this?

Reply:
no with hands its 2 rows